presto-parser/src/main/java/io/prestosql/sql/parser/ErrorHandler.java

```diff
@@ -182,6 +182,12 @@ public Analyzer(
                 int tokenIndex = current.tokenIndex;
                 CallerContext caller = current.caller;
 
+                while (stream.get(tokenIndex).getChannel() == Token.HIDDEN_CHANNEL) {
```

Comment:
I think this will fail if the trailing tokens are all whitespace -- it will run out of bounds

Reply:
I think EOF is always the last token and it's DEFAULT_CHANNEL (not HIDDEN_CHANNEL), so I don't need explicit bound check.
(Previously the code depended on this too, bounds were never checked explicitly.)

Reply:
I added one more test case: `"CREATE TABLE foo "` (the token stream ends with ...WS, EOF).

btw ANTLR emits only one WS token for all consecutive whitespace, so technically the `while` loop is not necessary. I am keeping it to avoid reader's astonishment.